tests/conformance/bindings/bindings.go

```diff
@@ -103,6 +104,13 @@ func startHTTPServer(url string) {
 }
 
 func (tc *TestConfig) createInvokeRequest() bindings.InvokeRequest {
+	if tc.ComponentType == "bindings" && tc.ComponentName == "kitex" {
```

Comment:
@ItalyPaleAle thoughts on this approach for conformance test?

Reply:
I think it's ok given that it's a unique case. Although maybe unnecessary given that there's a certification test?

Reply:
As discussed with @berndverst , we've dropped the conformance test since the same things were covered by the certification and unit tests. Thanks for submitting the PR, we're good to merge now :)